### ISSUE TITLE
[milight] iBox2 integration doesn't work in 8 channel mode

### ISSUE THREAD

**Issue by mirkoschubert** [openhab/openhab-addons#5277]
It's kind of a bug and a feature request...

## Expected Behavior

The Milight iBox2 (API v6) has usually 3 modes: 4, 8 or 99 channels.

* If you use the iBox2 only with the mobile app, only 4 channels are available.
* If you use the iBox2 with the [8 channel remote](https://www.amazon.com/Mi-Light-Wireless-Controller-Dimmable-Compatible/dp/B07J35734S/) (FUT089), you can setup 8 channels.
* There is actually a [99 channel remote](https://www.amazon.com/2-4GHz-Remote-Controller-Milight-FUT090/dp/B07NW4FQMW/) (FUT090) which unlocks 99 channels in the iBox 2, but no 3rd party supports it anyway.

I have the 8 channel remote, so I can control up to 8 channels with the app and the remote. For now I have assigned the first 4 channels to the Milight binding like in the documentation. I should be able to control those 4 channels (and more) just as well as in the 4 channel mode.

## Current Behavior

But I'm not. Channel 1 and 3 work perfectly. Channel 2 and 4 don't react to my commands in openHAB. The app and remote are working just fine.

The openhab.log reports turning the light on:

```
[ome.event.ItemCommandEvent] - Item 'Light_CR_Ceiling' received command ON
[nt.ItemStatePredictedEvent] - Light_CR_Ceiling predicted to become ON
[INFO ] [nal.protocol.MilightV6SessionManager] - Confirmation received for unsend command. Sequence number: 1
[INFO ] [nal.protocol.MilightV6SessionManager] - Confirmation received for unsend command. Sequence number: 1
```

...but the light doesn't turn on. I didn't check yet, but I suspect, that channel 1, 3, 5 and 7 should work, but 2, 4, 6 and 8 will not.

## Possible Solution

Before I bought the 8 channel remote, I wrote a node.js command line application to control the lights and simulate a sunset. So I tried it with the remote and got the same problem. Back then I used the `node-milight-promises` library, so I looked in the code and found out, that there were different functions particularly for the 8 channel mode, because apparently the 8 channel mode uses slightly different byte codes that the 4 channel mode.

So I fixed it in my node app and looked in the Milight binding code if there is anything similar. I don't know Java well, but I think it's not.

[Here](https://github.com/mwittig/node-milight-promise/blob/master/src/commandsV6.js) you can look up the codes for the 8 channel remote, if you want to fix it yourself. I obviously can't.

## Steps to Reproduce (for Bugs)

1. Buy a Milight iBox2, some lightbulbs and a 8 channel remote.
2. Connect the lightbulbs to the controller on the remote (and the app if you like).
3. Install the Milight binding and configure it as usual
4. Switch on one of the lights at a zone with an even number on the remote and openHAB ans see the difference.

## Context

See unter `Expected behavior`!

## Your Environment

* Version used: openHABian 2.4.0 and Milight Binding 2.4.0
* Environment name and version: Safari 12.1, Java 8
* Operating System and version: openHABian 2.4.0 on a Raspberry Pi 3 B

**Comment by davidgraeff** [comment]
You want to increase the "zones", I guess?
I'll make a PR for this, but I do not have all the required hardware here to test it, so we might need some iterations for it to work.

**Comment by RABUE** [comment]
I have tried to integrate my iBox2 v6 into OH3. 
The bridge and an installed RGBww are online. 
However, with no feedback on operation.
Operation via the Milight app and with a homebridge works fine.
A second bridge can also be installed in OH. 

After a few days of testing, I replaced the existing 8 channel remote FUT089 with the 4 channel remote FUT092 in the Milight app. 

After that, all the functions of 4 channels were given. 
Does anyone have a solution for the 8 channel control?

Unfortunately, I do not know how to program. Maybe it is possible to integrate the https://github.com/mwittig/node-milight-promise in OH3?

The node works fine with homebridge.

Thanks in advance
